data/network/src/commonMain/kotlin/social/androiddev/common/network/di/NetworkKoinModule.kt

```diff
@@ -12,18 +12,27 @@ package social.androiddev.common.network.di
 import io.ktor.client.HttpClient
 import io.ktor.client.plugins.defaultRequest
 import io.ktor.http.URLProtocol
+import org.koin.core.module.Module
+import org.koin.core.module.dsl.singleOf
+import org.koin.dsl.module
+import social.androiddev.common.network.MastodonApi
+import social.androiddev.common.network.MastodonApiKtor
 
-// TODO Update with Koin
-object NetworkDIModule {
+val networkModule: Module = module {
```

Comment:
Where do people stand on commenting publicly accessible API

Reply:
I personally agree with that. Something that I missed as I am racing to get some structure in place. Will update with javadocs on all public APIs